src/main/java/com/codeborne/selenide/webdriver/DriverFactory.java

```diff
@@ -12,7 +12,8 @@
 import java.io.File;
 
 public interface DriverFactory {
-  void setupWebdriverBinary();
+  default void setupWebdriverBinary() {
```

Comment:
Do we really need this method in interface?

Reply:
@rosolko  I was thinking that it's needed because some people might implement their own `DriverFactory`.
But now I see that we really can just remove this code. It would not be called anyway.